projects/bp-gallery/src/components/common/picture-gallery/BulkOperationsPanel.tsx

```diff
@@ -1,9 +1,9 @@
-import { Button, Icon } from '@mui/material';
+import { Button } from '@mui/material';
 import { FlatPicture } from '../../../types/additionalFlatTypes';
 import './BulkOperationsPanel.scss';
 
 export interface BulkOperation {
-  icon: string;
+  icon: JSX.Element;
```

Comment:
What do you think of using `ReactNode` here and everywhere else instead? It's a little more general and matches the type that is generally used for "stuff that can be rendered".

Reply:
A ReactNode can also return a string or a number, which isn't the default behavior I'd expect from an SVG Icon. Actually, setting the type to JSX.Element was really helpful in finding any leftover font icons, since those were strings and therefore threw an error.